### ISSUE TITLE
Add robots.txt parsing

### ISSUE THREAD

**Issue by joshua-bn** [spekulatius/PHPScraper#177]
Would be nice to have the ability to parse robots.txt like RSS feeds. `$web->robots`

https://github.com/bopoda/robots-txt-parser is a library. Not sure if it is the one to use here but it seems to do the job

**Comment by spekulatius** [comment]
Yeah, that's something to consider. I would opt for https://github.com/spatie/robots-txt instead as it's better maintained. What exactly do you want to achieve with the information?

**Comment by joshua-bn** [comment]
Personally, I am looking for sitemaps declared in robots.txt but I think there's also value in checking for rules for crawling.

**Comment by spekulatius** [comment]
Fair enough, that's definitely another use-case. I'll see how we can get
both working
